Index: vdr/vdr.c
===================================================================
--- vdr.orig/vdr.c	2012-03-14 19:58:53.000000000 +0100
+++ vdr/vdr.c	2012-03-14 19:59:25.000000000 +0100
@@ -173,6 +173,7 @@
   // Initiate locale:
 
   setlocale(LC_ALL, "");
+  setlocale(LC_NUMERIC, "C"); // makes sure any floating point numbers written use a decimal point
 
   // Command line options:
 
@@ -560,7 +561,6 @@
      isyslog("codeset is '%s' - %s", CodeSet, known ? "known" : "unknown");
      cCharSetConv::SetSystemCharacterTable(CodeSet);
      }
-  setlocale(LC_NUMERIC, "C"); // makes sure any floating point numbers written use a decimal point
 
   // Initialize internationalization:
 
