Description: Use ncursesw header file
Author: Tobias Grimm <tg@e-tobi.net>

Index: vdr/PLUGINS/src/skincurses/skincurses.c
===================================================================
--- vdr.orig/PLUGINS/src/skincurses/skincurses.c	2015-02-07 11:23:08.604231809 +0100
+++ vdr/PLUGINS/src/skincurses/skincurses.c	2015-02-07 11:23:08.604231809 +0100
@@ -6,7 +6,7 @@
  * $Id: skincurses.c 3.1 2014/01/05 10:56:27 kls Exp $
  */
 
-#include <ncurses.h>
+#include <ncursesw/ncurses.h>
 #include <vdr/osd.h>
 #include <vdr/plugin.h>
 #include <vdr/skins.h>
